flex/storages/rt_mutable_graph/schema.cc

```diff
@@ -646,63 +814,94 @@ static bool parse_edge_files(
       std::string file_format;
       std::string file_path;
       if (!get_scalar(files_node[i], "format", file_format)) {
+        LOG(ERROR) << "format is not set properly";
         return false;
       }
       if (file_format != "standard_csv") {
-        LOG(INFO) << "file_format is not set properly";
+        LOG(ERROR) << "file_format is not set properly";
```

Comment:
typically, in the error info, we should also output the provided `file_format`, and also give hints about supported one.